easybuild/easyconfigs/w/WHAM/WHAM-2.0.9.1-intel-2019a.eb

```diff
@@ -0,0 +1,41 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'MakeCp'
+
+name = "WHAM"
+version = "2.0.9.1"
+# Boltzmann’s constant is by default defined either in 'kcal/mol'
+# using other easyconfig with versionsuffix to distinguish change to 'kJ/mol'
+
+# No valid HTTPS - Let'sEncrypt Cert expired loong time ago
+homepage = 'http://membrane.urmc.rochester.edu/?page_id=126'
+description = """An implementation of WHAM: 
+the Weighted Histogram Analysis Method"""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
+
+# No valid https
+source_urls = ['http://membrane.urmc.rochester.edu/sites/default/files/wham/']
+sources = ['%(namelower)s-release-2.0.9.1.tgz']
+checksums = ['db790ff82742527356de91af7ff95a2d7f77cc554b15596552a85130edec87e7']
+
+# might be unnecessary
```

Comment:
I think we can drop the comment? 
```suggestion
```

Reply:
dropping make clean, easybuild should create new build dir each run.